presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -151,3 +151,161 @@ Hive, this table can be described in Presto::
++                 property name                  +                   type                   +               default value               +tuning the property                                           +
++================================================+==========================================+===========================================+==============================================================+
++ ``hive.assume-canonical-`` ``partition-keys``  +               ``Boolean``                +                 ``false``                 +Disable optimized metastore partition fetching for non-string +
++                                                +                                          +                                           +partition keys. Setting this property allows to avoid         +
++                                                +                                          +                                           +ignoring data with non-canonical partition values.            +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++   ``hive.domain-compaction-`` ``threshold``    +       ``Integer`` (at least ``1``)       +                  ``100``                  +Maximum number of ranges allowed in a tuple domain without    +
++                                                +                                          +                                           +compacting it. Higher value will cause more data              +
++                                                +                                          +                                           +fragmentation but allows to use row skipping feature when     +
++                                                +                                          +                                           +reading ORC data. Setting this value higher may have large    +
++                                                +                                          +                                           +impact on ``IN`` and ``OR`` clauses performance in scenarios  +
++                                                +                                          +                                           +making use of row skipping.                                   +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++      ``hive.force-local-`` ``scheduling``      +               ``Boolean``                +                 ``false``                 +Force splits to be scheduled on the same node as the Hadoop   +
```

Comment:
It looks like you are referring to a property explained earlier on this page under "Configuration Properties". Maybe instead of copy/paste we can simply refer the previous description?
Also, there is another related property (described elsewhere) that is worth mentioning here (node-scheduler.network-topology=flat). Brief characterization how they interact with each other and pros/cons are appropriate.